spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -230,9 +230,9 @@ def getCompletion(self, text_value):
       self.interpreterObject.setStatementsFinished(result, False)
 
 
-output = Logger()
-sys.stdout = output
-sys.stderr = output
+__zcStdOutput__ = __ZeppelinLogger__()
+sys.stdout = __zcStdOutput__
+sys.stderr = __zcStdOutput__
 
```

Comment:
I think you don't need to rename output, as it is not in the namespace of `repl`